templates/base.html

```diff
@@ -11,6 +11,7 @@
     <link href="{% sass_src 'sass/style.scss' %}" rel="stylesheet" type="text/css" />
     {% block style %}{% endblock %}
 </head>
+{% include '_navbar.html' %}
 <body>
     {% block body %}
     {% endblock %}
```

Comment:
cái này phải nằm trong thẻ body chứ
```suggestion
<body>
    {% include '_navbar.html' %}
    {% block body %}
    {% endblock %}
```

Reply:
Hic. Oke anh.